Lottie.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,6 +1,15 @@
 {
   "object": {
     "pins": [
+      {
```

Comment:
this is just because I didn't want to add `Equatable` conformance to the whole model tree